slack/slack_rtm.go

```diff
@@ -98,13 +99,14 @@ func NewSlackConnection(slackKey string) (*websocket.Conn, string) {
 }
 
 // GetMessage is a method for getting messages from the websocket connection
-func (slack *Slack) GetMessage() (m Message, err error) {
-	err = websocket.JSON.Receive(slack.Connection, &m)
+func (slack *Slack) GetMessage() Message {
+	var m Message
+	err := websocket.JSON.Receive(slack.Connection, &m)
 	if err != nil {
```

Comment:
I think we need to return this error and do this `if` in `bot.go`. Can you do it?